packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -389,10 +395,26 @@ function appendOutput(
 	node.outputs = node.outputs.filter((d) => d.value);
 
 	selectOutput(node, uuid);
-
+	generateSummary(node, outputPort);
```

Comment:
Because `generateSummary` is calling `updateOutputPort` I think there may be some race condition here when other nodes emit `update-output-port`

I will need to dig a bit deeper to be sure as its been a little bit since ive read this section

Reply:
The usage of update-output-port is frowned upon, I think we will probably shift base and store the summary outside as per other thread to side-step these issues altogether as per discussions in other threads.